custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -63,7 +86,10 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
             "sProcessing": "Loading. Please wait...",
         })
         .withOption('order', [[0, 'asc']])
-        .withDOM('ltipr');
+        .withDOM('ltipr')
+        .withOption('initComplete', function() {
```

Comment:
Should this be under FF?

Reply:
No. This will just connect the "See more" to the function defined in the controller.